From 6db4f2b3511874ce384d87ce7bbcdc0f4d21cf75 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marek.vasut+renesas@mailbox.org>
Date: Fri, 11 Jul 2025 23:23:50 +0200
Subject: [PATCH 8/9] FIXME: arm64: dts: renesas: r8a779g3: Select AVB0 VDDQ
 1V8 on Retronix R-Car V4H Sparrow Hawk EVTB1

The Retronix R-Car V4H Sparrow Hawk EVTB1 uses 1V8 IO voltage for
the AVB0 ethernet. Update the AVB0 pinmux settings to reflect the
change in IO voltage and correctly configure PFC POC7 "VDDQ18_25_AVB"
bit.

Signed-off-by: Marek Vasut <marek.vasut+renesas@mailbox.org>
---
 dts/upstream/src/arm64/renesas/r8a779g3-sparrow-hawk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/dts/upstream/src/arm64/renesas/r8a779g3-sparrow-hawk.dts b/dts/upstream/src/arm64/renesas/r8a779g3-sparrow-hawk.dts
index 04ae8e064c2..9406aba1683 100644
--- a/dts/upstream/src/arm64/renesas/r8a779g3-sparrow-hawk.dts
+++ b/dts/upstream/src/arm64/renesas/r8a779g3-sparrow-hawk.dts
@@ -448,6 +448,10 @@
 			drive-strength = <21>;
 		};
 
+		pins-vddq {
+			pins = "PIN_VDDQ_AVB0";
+			power-source = <1800>;
+		};
 	};
 
 	/* Page 28 / CANFD_IF */
-- 
2.34.1

